src/main/resources/digital/slovensko/autogram/server/server.yml

```diff
@@ -313,6 +313,73 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/BatchEndResponseBody"
+
```

Comment:
Neviem to zapichnut dobre, ale asi zosuladme uz aj verziu API s verziou autogramu.

Reply:
Asi ti nerozumiem, čo chceš kam zapichnúť a na tomto mieste v kóde už vôbec. Ale teda updatol som verziu api.